### PR TITLE
fix(chat): fix publish target path constructing (Issue #1853)

### PR DESCRIPTION
**Pull request by Alexander-Kezik**
**Description:**

fix publish target path constructing

Issues:

- #1853 

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1855.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat/src/components/Chat/Publish/PublishWizard.tsx [hunk 1]
```diff
@@ -234,9 +236,8 @@ export function PublishModal({
                       ? FeatureType.Chat
                       : FeatureType.Prompt,
                     trimmedPath,
-                    type === SharingType.ConversationFolder ||
-                      type === SharingType.PromptFolder
```
Comment (IlyaBondar) [inline apps/chat/src/components/Chat/Publish/PublishWizard.tsx:238]: should we do it for `PromptFolder`?

**Comment by dspashynskyi** [comment]
/deploy-review
>- **Deployment failed**: [failed](https://gitlab.deltixhub.com/Clients/deltix/nightly-test/dial/review/-/pipelines/501019)

**Comment by dspashynskyi** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1855.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by dspashynskyi** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1855.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by dspashynskyi** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1855.nightly-test.deltixhub.io
>E2E tests status: success

**Review by denys-kolomiitsev — DISMISSED** [review]

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1855.nightly-test.deltixhub.io
>E2E tests status: success

**Review by IlyaBondar — APPROVED** [review]

**Pull request merged**